index.html

```diff
@@ -0,0 +1,1059 @@
+      <li class="for-hide"><a href="javascript:void(0);">Team</a></li>
+      <li class="for-hide"><a href="javascript:void(0);">Advertise</a></li>
+      <li class="for-hide"><a href="javascript:void(0);">Jobs</a></li>
+      <li class="for-hide"><a href="javascript:void(0);">Contact</a></li>
+    </ul>
+
+    <ul class="main-nav">
+      <li class="hidden"><a href="javascript:void(0);">LATEST</a></li>
+      <li class="hidden"><a href="javascript:void(0);">HARD WORK</a></li>
+      <li class="hidden"><a href="javascript:void(0);">PLUGE D </a></li>
+      <li class="hidden"><a href="javascript:void(0);">FUNDAMENTAL</a></li>
+      <li class="hidden"><a href="javascript:void(0);">WORK 2030</a></li>
+      <li class="spacer"></li>
+      <li class="search">
+        <a href="">
+          <i class="material-icons">search</i>
+        </a>
+      </li>
+
+      <li class="menu-icon">
+        <a href="">
+          <span></span>
+          <span></span>
+          <span></span>
+        </a>
+      </li>
+    </ul>
+  </nav>
+  <main>
+    <section class="cover">
+      <h2 class="hidden-title">Cover news</h2>
+      <article class="article-big">
+        <img class="article-big-logo" src="https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/tnw.svg"
+          alt="The Next Web Logo" />
+        <div class="cover-article-info">
+          <p class="article-section">APPS</p>
+          <h2 class="article-title">
+            Instagram’s AI now warns you if your caption is offensive
+          </h2>
+          <p class="article-info">
+            by Rachel Kaser <i class="tiny material-icons">access_time</i> 1
+            hour ago <i class="tiny material-icons">share</i> 6
+          </p>
+        </div>
+      </article>
+      <article class="article-small-top article-small">
+        <div class="cover-article-info">
+          <p class="article-section">APPS</p>
+          <h2 class="article-title">
+            Instagram’s AI now warns you if your caption is offensive
+          </h2>
+          <p class="article-info">
+            by Rachel Kaser <i class="tiny material-icons">access_time</i> 1
+            hour ago <i class="tiny material-icons">share</i> 6
+          </p>
+        </div>
+      </article>
+      <article class="article-small-bottom article-small">
+        <div class="cover-article-info">
+          <p class="article-section">APPS</p>
+          <h2 class="article-title">
+            Instagram’s AI now warns you if your caption is offensive
+          </h2>
+          <p class="article-info">
+            by Rachel Kaser <i class="tiny material-icons">access_time</i> 1
+            hour ago <i class="tiny material-icons">share</i> 6
+          </p>
+        </div>
+      </article>
+    </section>
+
+    <section class="latest-news">
+      <div class="latest-news-headline">
+        <h2>
+          <a href="javascript:void(0)">
+            Latest news >
```

Comment:
Line number 120: Special characters must be escaped : [ > ]. Kindly recommended to delete symbol > after "Latest news"